ui/components/safe/SafeExecutionDisclaimer.tsx

```diff
@@ -0,0 +1,149 @@
+import { XMarkIcon } from '@heroicons/react/20/solid'
```

Comment:
There's a lot in here - kind of taking it on faith that you've tested sending funds via the team and it's all good. Looks fine here!